.gitpod.yml

```diff
@@ -21,7 +22,9 @@ tasks:
     yarn download:sample-traces &&
     yarn
   command: >
-    yarn start:electron ../TraceCompassTutorialTraces/ --hostname=0.0.0.0 --ignore-gpu-blacklist --no-sandbox
+    export GITPOD_URL=$(gp url) &&
+    export TRACE_SERVER_URL=$(echo "$GITPOD_URL" | sed -E "s/(http[s]:\/\/)(.*)/\18080-\2\/tsp\/api/") &&
+    yarn start:browser ../TraceCompassTutorialTraces/ --hostname=0.0.0.0 --ignore-gpu-blacklist --no-sandbox
```

Comment:
`--ignore-gpu-blacklist` and ` --no-sandbox` are Electron-specific and can be removed